internal/driver/webhtml.go

```diff
@@ -511,6 +615,11 @@ function dotviewer(nodes) {
     return null
   }
 
+  // convert a string to a regexp that matches that string.
+  function quotemeta(str) {
+    return str.replace(/([\\\.?+*\[\](){}])/g, '\\$1')
```

Comment:
Are we concerned about other characters like $ ^ |, or are they supposed to be a part of the regexp (then it would be hard to escape them as \$ would turn into \\$)?